tests/core/pyspec/eth2spec/test/helpers/rewards.py

```diff
@@ -61,6 +62,17 @@ def run_attestation_component_deltas(spec, state, component_delta_fn, matching_a
                 assert penalties[index] == 0
 
 
+def set_some_new_deposits(spec, state, rng):
+    num_validators = len(state.validators)
+    # last 10th of validators are new deposits
+    for i in range(len(state.validators))[0:num_validators // 10]:
+        index = num_validators - 1 - i
+        mock_deposit(spec, state, index)
+        # Set half to eligible for activation
+        if i % 2 == 0:
```

Comment:
```suggestion
        if rng.choice([True, False]):
```
This is what you intended? Why pass rng otherwise?

Reply:
`rng` over only 10 validators probably won't give you "half to eligible for activation".

Reply:
I want 10% of the vals to not be activated and half of them to be eligible for activation (in the activation queue)

Reply:
`rng` was in there before I changed the logic

I'll clean this up a bit and add better comments

Reply:
k, cleaner